tests/testsrc/tests/ecma/Array/15.4.4.3-1.js

```diff
@@ -31,8 +31,6 @@ writeHeaderToLog( SECTION + " Array.prototype.join()");
 var ARR_PROTOTYPE = Array.prototype;
 
 new TestCase( SECTION, "Array.prototype.join.length",           1,      Array.prototype.join.length );
-new TestCase( SECTION, "delete Array.prototype.join.length",    false,  delete Array.prototype.join.length );
```

Comment:
@gbrail has use a different approach for dealing with this in his commit
![image](https://github.com/user-attachments/assets/33d8d6c1-f98a-44d2-9068-68874c9e6ffc)

Reply:
Yeah, I will update this.

Reply:
Done.